readme.md

```diff
@@ -279,6 +280,33 @@ file.filePath() // ~/example.txt
 directory (slashed), if applicable, and suffixed with the (dotted) extension
 (if applicable).  Otherwise, an empty string is returned.
 
+### VFile#fileBasename()
+
+Get the filename, with extension, if applicable.
+
+**Example**
+
+```js
+var file = new VFile({
+  'directory': '~',
+  'filename': 'example',
+  'extension': 'txt'
+});
+
+String(file.fileBasename); // example.txt
+file.fileBasename() // example.txt
+```
+
+**Signatures**
+
+*   `string = vFile.fileBasename()`.
+
+**Returns**
+
+`string` — If the `vFile` has a `extension`, it will be prefixed with the
+`filename`, if applicable. If the `vFile` does not have `extension`,
+only `filename` will be returned. Otherwise, an empty string is returned.
+
 ### VFile#move(options)
```

Comment:
Maybe a simpler wording here is okay, for example, `` `string`— Returns the filePath without a directory, if applicable. Otherwise, an empty string is returned ``. Do you think that works?